packages/server/src/plugin/usageReporting/stats.ts

```diff
@@ -61,20 +61,30 @@ export class OurReport implements Required<IReport> {
     trace,
     asTrace,
     referencedFieldsByType,
+    // The max size a trace can be before it is sent as stats. Note that the
+    // Apollo reporting ingress server will never store any traces over 10mb
+    // anyway. They will instead be converted to stats as we would do here.
```

Comment:
I guess my caveat is that traces are always converted to stats too by the ingress — it's not an either/or.